lib/hypernova/blank_renderer.rb

```diff
@@ -31,4 +31,8 @@ def key
   def name
     job[:name]
   end
+
+  def id
+    SecureRandom.uuid
```

Comment:
I actually adjusted and wrote a spec [here](https://github.com/frozenfung/hypernova-ruby/commit/e0b8ad7496d418a5b5dfced30290cfcfc3e665db) feel free to cherry-pick into this PR (or don't 😄 )